src/Tribe/Field.php

```diff
@@ -340,6 +341,20 @@ public function doFieldName( $multi = false ) {
 		}
 
 		/**
+		 * returns the field's placeholder
+		 *
+		 * @return string the field value
+		 */
+		public function doFieldPlaceholder() {
```

Comment:
While I certainly like snake case better, I was making it match all it's neighbors. @borkweb what is your preference here?